File: iPad/VerBank-CSTSv3-IOSClientApi/ClientStation/api/csts/NetInfoNode.m
//
//  NetInfoNode.m
//  JEDIv7-CSTSv3-ClientAPI
//
//  Created by baolin on 13/5/16.
//  Copyright (c) 2013年 wangyubo. All rights reserved.
//

#import "NetInfoNode.h"

@implementation NetInfoNode
 

-(NSString *)getIpAddress{
    return ipAddress;
}

-(void)setIpAddress:(NSString *)_ipAddress{
    ipAddress=_ipAddress;
}

-(int)getPort{
    return port;
}

-(void)setPort:(int)_port{
    port=_port;
}

-(long long)getTime{
    return time;
}

-(void)setTime:(long long)_time{
    time=_time;
}

-(int)getType{
    return type;
}

-(void)setType:(int)_type{
    type=_type;
}

-(long)getLoginUserTime{
    return loginUseTime;
}

-(void)setLoginUserTime:(long)_loginUserTime
{
    loginUseTime=_loginUserTime;
}

-(int)getReason{
    return reason;
}

-(void)setReason:(int) _reason{
    reason=_reason;
}

-(NSMutableArray *)getReconnectVec{
    return reconnectVec;
}

-(void)setReconnectVec:(NSMutableArray *)_reconnectVec{
    reconnectVec=_reconnectVec;
}
@end

File: WeChatService/EX02-CSTSv3-ClientAPI/src/jedi/ex01/client/station/api/CSTS/NetInfoNode.java
package jedi.ex01.client.station.api.CSTS;

public class NetInfoNode {

	public static final int TYPE_LOST=1;
	public static final int TYPE_CONNECT=2;
	
	public static final int REASON_NETLOST=1;
	public static final int REASON_TIMEOUT=2;
	
	private int type;
	private String ipAddress;
	private int port;
	private long time;
	private long loginUseTime;
	private int reason;
	private APIReConnectInfoNode[] reconnectVec;

	public String getIpAddress() {
		return ipAddress;
	}

	public void setIpAddress(String ipAddress) {
		this.ipAddress = ipAddress;
	}

	public int getPort() {
		return port;
	}

	public void setPort(int port) {
		this.port = port;
	}

	public long getTime() {
		return time;
	}

	public void setTime(long time) {
		this.time = time;
	}

	public int getType() {
		return type;
	}

	public void setType(int type) {
		this.type = type;
	}

	public long getLoginUseTime() {
		return loginUseTime;
	}

	public void setLoginUseTime(long loginUseTime) {
		this.loginUseTime = loginUseTime;
	}

	public int getReason() {
		return reason;
	}

	public void setReason(int reason) {
		this.reason = reason;
	}

	public APIReConnectInfoNode[] getReconnectVec() {
		return reconnectVec;
	}

	public void setReconnectVec(APIReConnectInfoNode[] reconnectVec) {
		this.reconnectVec = reconnectVec;
	}

}
